src/main/java/org/ibp/api/java/impl/middleware/ontology/validator/VariableValidator.java

```diff
@@ -515,9 +535,9 @@ void variableShouldBeEditable(final VariableDetails variable, final Errors error
 			final boolean propertyEqual = Objects.equals(propertyId, oldVariable.getProperty().getId());
 			final boolean methodEqual = Objects.equals(methodId, oldVariable.getMethod().getId());
 			final boolean scaleEqual = Objects.equals(scaleId, oldVariable.getScale().getId());
-			final boolean minValuesEqual = editableVariable ? true : StringUtil.areBothEmptyOrEqual(variable.getExpectedRange().getMin(), oldVariable.getMinValue());
-			final boolean maxValuesEqual = editableVariable ? true : StringUtil.areBothEmptyOrEqual(variable.getExpectedRange().getMax(), oldVariable.getMaxValue());
-			final boolean aliasEqual = editableVariable ? true : StringUtil.areBothEmptyOrEqual(variable.getAlias(), oldVariable.getAlias());
+			final boolean minValuesEqual = editableVariable ? editableVariable : StringUtil.areBothEmptyOrEqual(variable.getExpectedRange().getMin(), oldVariable.getMinValue());
```

Comment:
why this change is required for the file data type limitation? 
or was it a bug? if it was a bug, then should be included to be tested
@cuenyad

Reply:
Yes, it's gonna be tested. 
When I added the restricction made a SonaLint cleanUp.